src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+h1 {
+  display: none;
+}
+
+.calendar {
+  margin: 40px auto 0;
+  width: ($cell-size + 1) * $coll-couny;
```

Comment:
There is a typo in the variable name. It should be $col-count instead of $coll-couny